scoop/src/main/java/com/lyft/scoop/ViewController.java

```diff
@@ -10,10 +10,12 @@
     private boolean attached;
     private Scoop scoop;
     private View view;
+    private ViewBinder viewBinder;
 
     final void attach(View view) {
         this.view = view;
-        Scoop.viewBinder.bind(this, view);
+        this.viewBinder = scoop.viewBinderFactory.create(this);
```

Comment:
nicer if `Scoop` exposes a `createViewBinder` method instead of `viewBinderFactory`